SUMMER/SUMMER.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using Eclr;
+using System.Iec61131Lib;
+using System.Runtime.InteropServices;
+
+
+
+namespace SUMMER
+{
+    /***    Start of Managed Unit Conversion Blocks   ***/
+    [FUNCTION_BLOCK]
+    [Managed]
```

Comment:
Why about Native code.

Reply:
Do we need to develop Managed code or Native here?